utils_cv/detection/data.py

```diff
@@ -132,3 +140,339 @@ def coco_labels() -> List[str]:
         "hair drier",
         "toothbrush",
     ]
+
+
+def extract_masks_from_labelbox_json(
+    labelbox_json: Union[str, Path],
+    old_dir: Union[str, Path],
+    new_dir: Union[str, Path] = None,
+) -> None:
+    """ Extract masks from Labelbox annotation JSON file.
+
+    The annotation files in
+    [odFridgeObjects](https://cvbp.blob.core.windows.net/public/datasets/object_detection/odFridgeObjects.zip)
+    are in the format of PASCAL VOC:
```

Comment:
Don't think we need to show again here what PascalVOC format looks like. Instead can simply write that the 01 notebook shows what the PascalVOC format looks like.

Reply:
Removed